src/policy/v3_policy.h

```diff
@@ -15,8 +15,9 @@
 #include <set>
 #include <string>
 
-// This module enforces rules for transactions with nVersion=3 ("v3 transactions") which help make
+// This module enforces rules for transactions BIP 431 TRUC transactions (with nVersion=3) which help make
```

Comment:
There is either too many or too few words here:
```suggestion
// This module enforces rules for BIP 431 TRUC transactions (with nVersion=3) which help make
```

Reply:
fixed, thanks